__tests__/app.test.js

```diff
@@ -138,6 +137,28 @@ describe("/api", () => {
             });
         });
       });
+      describe("/comments", () => {
+        describe("GET", () => {
+          test("200 - recieves back a list of comments on the article", () => {
+            return request(app)
+              .get("/api/articles/1/comments")
+              .expect(200)
+              .then(({ body: { comments } }) => {
```

Comment:
would like to see a comments.length check to make sure you're getting the right number of comments back